roles/reproducer/molecule/crc_layout/converge.yml

```diff
@@ -91,6 +91,20 @@
             <ip family='ipv4' address='{{ _networks.osp_trunk.range | ansible.utils.nthhost(1) }}' prefix='24'>
             </ip>
           </network>
+    cifmw_deploy_va_steps:
```

Comment:
I this variable consumed as part of this PR? Cause I think it's part of the next ones and it may be part of those ones.

Reply:
actually, that one isn't needed anymore. Have to double-check, but molecule would load the architecture content directly.

Reply:
useless, removed.